homeworks/edgar_harutyunyan/homeworks/MyHomework/Homework9/Employee.java

```diff
@@ -0,0 +1,39 @@
+public class Employee{
+	public static void main(String[] args){
+     Employees employess = new Employee("");
+     System.out.println(Employee.name);
+	}
+class Employees{
+	int name;
+	int yearofjoining;
+	int address;
+
+	Employees(int name, int yearofjoining , int address){
+		this.name = Robert;
+		this.yearofjoining = 1994 ;
+		this.address = 64C-WallsStreat ;
+	}
+    class EmployeesS{
```

Comment:
why you create class for each employee? You should have just one class Employee and define class objects with different values.